### ISSUE TITLE
[TF1.12][graph_transforms] The INT8 performance test of graph_transforms for inceptionv3 model, GPU utilization extremely low.

### ISSUE THREAD

**Issue by oscarriddle** [tensorflow/tensorflow#24033]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): CentOS7
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: No
- TensorFlow installed from (source or binary): source 
- TensorFlow version (use command below): 1.12
- Python version: 2.7
- Bazel version (if compiling from source): 0.15
- GCC/Compiler version (if compiling from source): 4.9.3
- CUDA/cuDNN version: 9.0/7.0.5
- GPU model and memory: TitanXP

**Describe the current behavior**
I prepared an inception v3 model and try to quantify it by the tool "graph_transforms".
This is my conversion command:
`bazel-bin/tensorflow/tools/graph_transforms/transform_graph --in_graph=frozen_graph.pb --out_graph=out_graph.pb --inputs='Placeholder:0' --outputs='InceptionV3/Logits/SpatialSqueeze:0' --transforms='add_default_attributes strip_unused_nodes(type=float, shape="16,299,299,3") remove_nodes(op=Identity, op=CheckNumerics) fold_constants(ignore_errors=true) fold_batch_norms fold_old_batch_norms quantize_weights quantize_nodes strip_unused_nodes sort_by_execution_order'`

Then I evaluated the accuracy loss by test dataset which is consists of ~80k images. The result shows the accuracy decreased from 70.1% to 64.03%.

Then I try to evaluate the GPU performance by repeatedly run a batch of images, and the GPU utilization is extremely low and the speed is very slow. 
```
$ nvidia-smi dmon -i 0
# gpu   pwr  temp    sm   mem   enc   dec  mclk  pclk
# Idx     W     C     %     %     %     %   MHz   MHz
    0    59    35     2     0     0     0  5508  1430
    0    59    35     0     0     0     0  5508  1430
    0    59    36     2     0     0     0  5508  1430
    0    59    36     0     0     0     0  5508  1430
    0    59    36     2     0     0     0  5508  1430
    0    59    36     2     0     0     0  5508  1430
    0    59    36     0     0     0     0  5508  1430
    0    59    36     3     0     0     0  5508  1430
    0    59    36     0     0     0     0  5508  1430
    0    59    36     2     0     0     0  5508  1430
    0    59    36     0     0     0     0  5508  1430
    0    60    36     2     0     0     0  5508  1430
    0    59    36     4     0     0     0  5508  1430
    0    59    36     0     0     0     0  5508  1430
    0    60    36     3     0     0     0  5508  1430
    0    59    37     0     0     0     0  5508  1430
    0    59    37     2     0     0     0  5508  1430
    0    59    37     2     0     0     0  5508  1430
    0    59    36     2     0     0     0  5508  1430
    0    59    37     3     0     0     0  5508  1430
    0    59    37     0     0     0     0  5508  1430
    0    59    37     3     0     0     0  5508  1430
    0    59    37     0     0     0     0  5508  1430
    0    59    37     2     0     0     0  5508  1430
    0    59    37     4     0     0     0  5508  1430
```

There shall exists some bugs in this tool, which is related to the GPU utilization.

Any idea will be welcome.

**Comment by oscarriddle** [comment]
Below is the GPU utilization info when running on original frozen model.

**Comment by tatianashp** [comment]
Is this issue still relevant? There has been no activity since early December.
